packages/scene-composer/src/components/StateManager.tsx

```diff
@@ -157,7 +167,16 @@ const StateManager: React.FC<SceneComposerInternalProps> = ({
           },
         ];
       }
-
+      // Add entityID info part of additional component data
+      // We assumed IDataBindingMap will have only one mapping as data binding
+      // will always have only one entity data.
+      if (entityBindingComponent) {
+        additionalComponentData?.push({
+          dataBindingContext: !entityBindingComponent?.valueDataBindings?.[0].valueDataBinding?.dataBindingContext
+            ? undefined
+            : extractEntityId(entityBindingComponent?.valueDataBindings?.[0].valueDataBinding),
```

Comment:
I think data returned here should be consistent with the actual entity binding type - if the entity binding data format support multiple, then we should return array here even though the array may only have 1 element. or we make the binding data format be single binding only, then we just return that. 

otherwise, if in the future we can having multiple entity binding for some reason, we will need to change the return value type here

Reply:
We don't support multiple entity binding data. one scene node/ 3d objects can have only one entity data binding. I think it's not necessary to return array objects if this has only one binding.